docs/projects/ad4134_fmc/index.rst

```diff
@@ -0,0 +1,252 @@
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`AD4134` is a quad channel, low noise, simultaneous sampling, precision 
+analog-to-digital converter (ADC), based on the continuous time sigma-delta 
+(CTSD) modulation scheme. This architecture inherently rejects signals around
+the ADC aliasing frequency band, giving the device its inherent antialiasing 
+capability, and removesthe need for a complex external antialiasing filter.
+
+This device has four independent converter channels in parallel, each with a CTSD 
+modulator and a digital decimation and filtering path. It enables simultaneous 
+sampling of four signal sources, with a maximum input bandwidth of 391.5 kHz.
+It supports a wide range of ODR frequencies, from 0.01 kSPS to 1496 kSPS wih less
+than 0.01 SPS adjustment resolution, allowing the user to granularly vary sampling
+speed to achieve coherent sampling.
+
+The :adi:`AD4134` supports two device configuration schemes: serial peripheral interface
+(SPI) and hardware pin configuration (pin control mode). The SPI control mode offers
+access to all the features and configuration options available on the chip.
+Pin control mode offers the benefit of simplifying the device configuration, enabling
+the device to operate autonomously after power-up operating in a standalone mode.
+
+The HDL reference design for the EVAL-AD4134 provides all the interfaces that are 
+necessary to interact with the device using a Xilinx FPGA development board; to 
+acquire continuous data from the 24-bit 4-channel precision alias free ADC device.
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD4134 <EVAL-AD4134>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD4134`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+-  :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on FMC slot
+
+Block design
+-------------------------------------------------------------------------------
+
+The reference design uses the SPI Engine Framework to interface with the AD4134 ADC
+and only supports the slave mode with both DCLK and ODR generated by the FPGA.
+The device sends data on the 4 DIN bits.
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The data path and clock domains are depicted in the below diagrams:
+
+.. image:: ad4134_hdl.svg
+   :width: 800
+   :align: center
+   :alt: AD4134-FMC/ZED block diagram
+
+Jumper setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+================== ========= ====================================
+Jumper/Solder link Position  Description
+================== ========= ====================================
+JP16	             Mounted	 MODE (Slave) and DCLKIO (Input)
```

Comment:
Fix indentation inside the table and make sure the === symbol from each header has the same length with the text underneath it.